build.gradle

```
@@ -48,11 +47,4 @@ jar {
 	from {
 		configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
 	}
-}
-
-//distZip {
-//	into(project.name+"-"+getVersion()) {
-//		from '.'
-//		include 'conf/*'
-//	}
-//}
+}
```

newline at end of file

```suggestion
}

```